credentialz/credentialz.proto

```diff
@@ -442,12 +443,30 @@ message UserPolicy {
     repeated Option options = 2;
   }
 
+  message AuthorizedPrincipalCheck {
+    enum PrincipalCheckTool {
```

Comment:
enum Tool {
  TOOL_UNSPECIFIED =  0;
  TOOL_HIBA = 1; 
}

Reply:
+1 (not that we have a choice ;-) )